src/components/JobDetailCard.jsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled';
+
+// Temporarily importing JSON here to test code
+import data from '../assets/data.json';
+
+const JobDetailContainer = styled.div`
+  padding: ${props => props.theme.layout.space500};
+  margin: 0 auto;
+  border-radius: 6px;
+  background-color: ${props => props.theme.colors.white};
+  max-width: 730px;
+`;
+
+const JobDetailHeader = styled.div`
+  color: ${props => props.theme.colors.darkGray};
+  display: flex;
+  align-items: center;  
+  justify-content: space-between;
+`;
+
+const JobTitle = styled.h1`
+  color: ${props => props.theme.colors.darkBlue};
+  margin-top: ${props => props.theme.layout.space50};
+  margin-bottom: ${props => props.theme.layout.space100};
+`;
+
+const JobLocation = styled.div`
+  font-size: ${props => props.theme.typography.fs87};
+  font-weight: ${props => props.theme.typography.fwBold};
+  color: ${props => props.theme.colors.violet};
+`;
+
+const JobDetailDescription = styled.div`
+  p, ul, ol {
+    color: ${props => props.theme.colors.darkGray};
+  }
+`;
+
+export const JobDetailCard = () => {
+
+  // TODO: For testing. Remove this once the components are connected
+  const job = data[0];
```

Comment:
I love your approach to testing the component!

Let's replace the variables below from the JSON data with prop values, as that's how we will pass data to this component. Let me know if you have any questions about this.

Reply:
I updated it to be a job object, does that work? Or do you mean individual prop values?

`const JobDetailCard = ({ job })`

Reply:
@eileenxue my bad for not being clearer earlier... 😓 Can you update the props to pass in the individual values such as the following: 
-  postedAt
- position
- location
- description
- requirements
- role

Reply:
Okay great, just updated. 😀